### PR TITLE
JC-2210 Dummy image is not loaded in firefox

### PR DESCRIPTION
**Pull request by drgremlin**

### CODE REVIEW

**Comment by ctapobep** [comment]
@skythet would you be able to review the PR?
@drgremlin the issue should be JC-2210, not JC2210, could you please amend the commit message?

File: jcommune-service/src/main/resources/kefirbb.xml [hunk 1]
```diff
@@ -254,6 +255,11 @@ get XSS attacks.</p>-->
         <template><![CDATA[<a title="" href="]]><var name="addr"/><![CDATA[" class="pretty-photo"><img class="thumbnail" alt="" src="]]><var name="addr"/><![CDATA[" onError="imgError(this)" /></a>]]></template>
     </code>
 
+    <code name="img2" priority="2">
+        <pattern ignoreCase="true">[img]<var name="addr" regex=""/>[/img]</pattern>
```
Comment (skythet) [inline jcommune-service/src/main/resources/kefirbb.xml]: var tag not required, `[img][/img]` sufficiently

**Comment by ctapobep** [comment]
@drgremlin will you update the PR?

**Comment by drgremlin** [comment]
PR updated

**Comment by ctapobep** [comment]
Ok, could you please squash these 2 commits into 1?

**Comment by ctapobep** [comment]
Duplicate of #190

**Pull request closed without merging**